src/rank_llm/rerank/prompt_templates/rank_fid_score_template.yaml

```diff
@@ -1,3 +1,6 @@
 method: "rankfid"
 query: "question: {query}"
 text: "question: {query} context: {passage}"
+output_patterns: 
+  - r"^\[\d+\]( > \[\d+\])*$" # pattern for raw output
```

Comment:
Here and every where else, do not rely on your comments and indexes. Preserve the dictionary format, and get rid of the output_patterns:
"output_validation_regex": 
"output_extraction_regex":